lib/common/zstd_internal.h

```diff
@@ -137,7 +137,10 @@ static const U32 OF_defaultNormLog = 5;
 /*-*******************************************
 *  Shared functions to include for inlining
 *********************************************/
-static void ZSTD_copy8(void* dst, const void* src) { memcpy(dst, src, 8); }
+static void ZSTD_copy8(void* dst, const void* src)
+{
+    MEM_write64(dst, MEM_read64(src));
```

Comment:
While I understand it does better follow "mem.h",
I'm wondering in which case this construction is better ?